test/Maqduni.AspNetCore.Identity.RavenDb.Tests/TestStores.cs

```diff
@@ -111,6 +111,49 @@ public void RoleCreateAsync(string roleName)
             Assert.True(result.Succeeded);
         }
 
+        [Theory(DisplayName = "Role AddClaimsAsync")]
+        [InlineData("Admin", "GraduatedSchoolYear", "2006")]
+        [InlineData("Admin", "GraduatedUniversityYear", "2010")]
+        public void RoleAddClaimsAsync(string roleName, string claimType, string claimValue)
+        {
+            var role = _roleStore.FindByNameAsync(roleName).Result;
+            Assert.NotNull(role);
+
+            var claim = new Claim(claimType, claimValue);
+            _roleStore.AddClaimAsync(role, claim).Wait();
+
+            var claims = _roleStore.GetClaimsAsync(role).Result;
+            Assert.True(claims.Any(c => c.Type.Equals(claim.Type, StringComparison.OrdinalIgnoreCase) && c.Value.Equals(claim.Value, StringComparison.OrdinalIgnoreCase)));
+
+            var result = _roleStore.UpdateAsync(role).Result;
+            Assert.True(result.Succeeded);
+        }
+
+        [Theory(DisplayName = "Role WithClaimsRemoveClaimAsync")]
+        [InlineData("Admin", "GraduatedSchoolYear", "2006")]
+        [InlineData("Admin", "GraduatedUniversityYear", "2010")]
+        public void RoleWithClaimsRemoveClaimAsync(string roleName, string claimType, string claimValue)
```

Comment:
Please rename the function to RoleClaimsRemoveClaimAsync. A agree that you are implying adding and removing claims, but I think it'd be better to have a unified naming pattern for all functions. I like the idea of adding and removing claims, and I can implement it in all of the tests.